libcaf_core/src/logger.cpp

```diff
@@ -146,6 +249,13 @@ logger::line_builder& logger::line_builder::operator<<(const std::string& str) {
   return *this;
 }
 
+logger::line_builder& logger::line_builder::operator<<(string_view str) {
+  if (!str_.empty() && str_.back() != ' ')
```

Comment:
Why does this check the last character while the overload below does not?

Reply:
Good question. I fixed the other implementations with c083daa.